frontstage/controllers/party_controller.py

```diff
@@ -356,13 +355,6 @@ def caching_data_for_survey_list(cache_data, surveys_ids, business_ids, tag):
     # in the cache_data.
     threads = []
 
-    for survey_id in surveys_ids:
```

Comment:
I think with this, the surveys_ids variable isn't used.  can you remove it from this function and the thing that calls it?